k8s/README.md

```diff
@@ -242,143 +263,74 @@ aws ecr create-repository \
+Work in progess...
 
-Now that you have a zqd running with a space, and you have made an archive from zar data, you can use zapi commands to query the sample data. Example:
+For now just use:
 ```
-zapi -s http-space-1 get "head 1"
-zapi -s http-space-1 get "tail 1"
+export POD_NAME=$(kubectl get pods --namespace zq -l "app.kubernetes.io/name=zqd,app.kubernetes.io/instance=zqd-test-1" -o jsonpath="{.items[0].metadata.name}")
+kubectl --namespace zq port-forward $POD_NAME 9867:9867
 ```
-The zapi commands in quotes are the same as Brim queries. Notice that "head 1" runs much faster than "tail 1", which has to read more data from s3.
 
-## Deploy zqd into the local cluster with helm
-The helm 3 chart is in k8s/charts/zqd. To install zqd with the helm chart:
-```
-helm upgrade zqd-test-1 charts/zqd --install
-```
-This will install the chart if it is not yet present, or upgrade it to latest if it is installed. (The helm chart adds a timestamp as an annotation to the deployment, so it will always restart the pods.)
+## Setting up an EC2 instance to run zar import
 
-You can confirm the pod has started with:
-```
-kubectl get pod
-```
-And view Helm installs with:
-```
-helm ls
-```
-## Testing connectivity to zqd
-A simple test for zqd is to send a /status request to its http endpoint. This is used for the K8s liveness probe. Substitute the pod id into a port-forward command:
+This is for importing large zeek logs from S3 to a zar archive also on S3.
 
+Create an EC2 instance in the same region as the S3 buckets you want to access. SSH into the instance, and:
 ```
-kubectl get pod
-kubectl port-forward zqd-test-1-66b5f9dc8-8dshh 9867:9867
+# provide your credentials
+aws configure
+# make sure you can see the S3 buckets
+aws s3 ls
+# install golang tools
+wget https://golang.org/dl/go1.14.7.linux-arm64.tar.gz
+sudo tar -C /usr/local -xzf go1.14.7.linux-arm64.tar.gz
+echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bash_profile
+source ~/.bash_profile
+go version  # make sure go is there!
+# install git, clone and install zq
+sudo yum install git -y
+git clone https://github.com/brimsec/zq
+make install
+~/go/bin/zar help  # make sure zar is built!
 ```
-And in another console, use curl:
+Now you have an environment where you can run zar and it can operate on data in S3 with good bandwidth. Example zar command:
 ```
-curl -v http://localhost:9867/status
+~/go/bin/zar import -R s3://zqd-demo-1/mark/zeek-logs/dns s3://brim-sampledata/wrccdc/zeek-logs/dns.log.gz
 ```
-You should get an 'ok' response.
 
+## [Trouble shooting](trouble-shooting.md)
 
-# Tearing down the environment
+## Tearing down the environment
```

Comment:
Would you please move this section up near the other local cluster work, so that its not separated by the AWS/EKS instructions?

Reply:
Also, we should eksctl teardown instructions.

Reply:
Yes on both